### ISSUE TITLE
Grader runtime error 0042

### ISSUE THREAD

**Issue by NestZ** [programming-in-th/grader#25]
Source code:
```
#include<bits/stdc++.h>
using namespace std;
typedef long long int ll;
int main(){
	int n;cin >> n;
	while(n--){
		int t;cin >> t;
		long double l = 1.0;
		while(t--)l *= 2.0;
		printf("%.0Lf\n", l);
	}
}
```

**Comment by pongsaphol** [comment]
Grader has been killed when the memory of the task is not greater than 8 MB. Now, I changed the memory limit of this task to 16 MB but this issue won't be closed.

**Comment by pongsaphol** [comment]
@szawinis will investigate this issue.

**Comment by galletas1712** [comment]
Fixed in #27
